tests/policy_engine/resources/policies/valid/slsa-verifier.dl

```diff
@@ -5,4 +5,4 @@
 #include "prelude.dl"
 
 Policy("auth-provenance", repositoryid, "") :- check_passed(repositoryid, "mcn_provenance_level_three_1").
-apply_policy_to("auth-provenance", repo_id) :- is_repo(repo_id, "github.com/slsa-framework/slsa-verifier", _).
+apply_policy_to("auth-provenance", repo_id) :- is_component(repo_id, "pkg:github.com/slsa-framework/slsa-verifier@fc50b662fcfeeeb0e97243554b47d9b20b14efac").
```

Comment:
Should this be `component_id` instead of `repo_id`?

Reply:
See commit [efd2cf5](https://github.com/oracle/macaron/pull/473/commits/efd2cf5526b794814260da562e620f756d9bd007)